source/Halibut.Tests/Transport/SecureClientFixture.cs

```diff
@@ -63,13 +62,11 @@ public async Task SecureClientClearsPoolWhenAllConnectionsCorrupt(SyncOrAsync sy
                 Params = new object[] { "Fred" }
             };
 
-            var secureClient = new SecureListeningClient(GetProtocol, endpoint, Certificates.Octopus, log, connectionManager);
+            var secureClient = new SecureListeningClient((stream, logger) => GetProtocol(stream, logger), endpoint, Certificates.Octopus, log, connectionManager);
             ResponseMessage response = null!;
 
 #pragma warning disable CS0612
-            await syncOrAsync
-                .WhenSync(() => secureClient.ExecuteTransaction((mep) => response = mep.ExchangeAsClient(request), CancellationToken.None))
-                .WhenAsync(async () => await secureClient.ExecuteTransactionAsync(async (mep, ct) => response = await mep.ExchangeAsClientAsync(request, ct), CancellationToken.None));
```

Comment:
Why did this get deleted?

Reply:
It shouldn't have, test fixed up to work for sync and async